samples/bluetooth/fast_pair/locator_tag/src/main.c

```diff
@@ -90,8 +90,6 @@ static void fmdn_factory_reset_prepare(void)
 	/* Disable advertising request from the UI. */
 	fp_adv_ui_request = false;
 	app_fp_adv_request(&fp_adv_trigger_ui, fp_adv_ui_request);
-
-	app_fp_adv_rpa_rotation_suspend(false);
```

Comment:
Is it okay, to remove it here and not call it inside fp_adv module on factory_rest_prepare callback?